mon-pix/app/components/feedback-panel.js

```diff
@@ -1,51 +1,23 @@
 import { isEmpty } from '@ember/utils';
 import $ from 'jquery';
-import { equal } from '@ember/object/computed';
 import { inject as service } from '@ember/service';
 import Component from '@ember/component';
 import config from 'mon-pix/config/environment';
 
-const FORM_CLOSED = 'FORM_CLOSED';
-const FORM_OPENED = 'FORM_OPENED';
-const FORM_SUBMITTED = 'FORM_SUBMITTED';
-
 export default Component.extend({
 
-  store: service(),
-
   classNames: ['feedback-panel'],
 
+  store: service(),
+
   assessment: null,
   challenge: null,
-  collapsible: true,
 
-  _status: FORM_CLOSED,
   _content: null,
   _error: null,
 
-  isFormClosed: equal('_status', FORM_CLOSED),
-  isFormOpened: equal('_status', FORM_OPENED),
-  isFormSubmitted: equal('_status', FORM_SUBMITTED),
-
-  didReceiveAttrs() {
-    this._super(...arguments);
-    this._reset();
-  },
-
-  _reset() {
-    this.set('_content', null);
-    this.set('_error', null);
-    this.set('_status', this._getDefaultStatus());
-  },
-
-  _closeForm() {
-    this.set('_status', FORM_CLOSED);
-    this.set('_error', null);
-  },
-
-  _getDefaultStatus() {
-    return this.collapsible ? FORM_CLOSED : FORM_OPENED;
-  },
+  openForm: false,
```

Comment:
Ce boolean n'est pas très bien nommé. `formOpened`, `isFormOpened`, ...

Reply:
C'est un booleen pour activer quelque chose, c'est plus dans le sens "should open form" que "form is opened", du coup j'ai pas trop d'idée mieux.

Reply:
Le renommage en `isFormOpened` me paraîtrait bien marcher.
Dans le toogle, ça donnerait : 
```
this.set('isFormOpened', false);
this.set('isSubmitted', false);
```